adoorback/account/views.py

```diff
@@ -1091,9 +1093,26 @@ def get_queryset(self):
         sorted_friends = sorted(mutual_friends_count_dict.items(), key=lambda x: x[1], reverse=True)[:25]
         sorted_friend_ids = [friend_id for friend_id, _ in sorted_friends]
 
-        # Add 10 random users
+        # Add 10 random users who are experiment participants, not just random users
+        participant_emails = set()
+        csv_path = os.path.join(settings.BASE_DIR, 'adoorback', 'assets', 'user_list.csv')
```

Comment:
```suggestion
        csv_path = os.path.join(settings.BASE_DIR, 'assets', 'user_list.csv')
```
I just tested the code and this line needs to be changed like this to make it work, since `BASE_DIR` is already pointing to `adoorback/adoorback`.